sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/ExecutorServiceParallelExecutor.java

```diff
@@ -368,25 +372,25 @@ private boolean shouldShutdown() {
     }
 
     private void mightNeedMoreWork() {
```

Comment:
Can you add Javadoc on what "mightNeedMoreWork" does/means? It is neither verby (describing what it does) nor noun-y (describing what it returns).